_DeploymentAndDistroPackaging/ansible/README.md

```diff
@@ -0,0 +1,33 @@
+# CIAO Deployment with Ansible Playbooks
+
+* Work in Progress
+
+## Prework
+
+ - Deploy SSH key from the host running ansible, to all hosts in the cluster setup.
+ - Install Ansible and his dependencies.
+
```

Comment:
Some information that may be useful:
- How the ansible package is named at Ubuntu, ClearLinux, etc